addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -1030,6 +1030,199 @@ There is no trigger anymore but one can use transformations to accomplish the sa
+
+```
+Thing data wago_s1_000 [ readStart="12288", readValueType="bit", writeStart="12288", writeValueType="bit", writeType="coil" ]
+```
+
+The second Item of the 1.x binding (offset `1`) is defined as follows.
+
+```
+Switch BarSwitch  "Bar Switch" {modbus="slave1:1"}
+```
+
+This leads to the thing definition
+
+```
+Thing data wago_s1_001 [ readStart="12289", readValueType="bit", writeStart="12289", writeValueType="bit", writeType="coil" ]
+```
+
+Note the absolute address `12289` (12288+1) which has to be used here.
+
+Incorporating this definitions into the thing file leads to:
+
+`wago.things`:
+
+```
+Bridge modbus:tcp:wago [ host="192.168.2.9", port=502, id=1 ] {
+
+    Bridge poller wago_slave1 [ start=12288, length=128, refresh=500, type="coil" ] {
+        Thing data wago_s1_000 [ readStart="12288", readValueType="bit", writeStart="12288", writeValueType="bit", writeType="coil" ]
+        Thing data wago_s1_001 [ readStart="12289", readValueType="bit", writeStart="12289", writeValueType="bit", writeType="coil" ]
+    }
+
+    Bridge poller wago_slave2 [ start=12338, length=100, refresh=4000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave3 [ start=12438, length=100, refresh=5000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave4 [ start=12538, length=100, refresh=10000, type="holding" ] {
+    }
+}
+```
+
+Save this in the `things` subdirectory of your openHAB 2 config. 
+Watch the file `events.log` as it lists your new added `data` `Things`. 
+Given that there are no config errors, they quickly change from `INITIALIZING` to `ONLINE`. 
+
+Finally the Item definition has to be changed to refer to the new created `data` `Thing`. 
+You can copy the names you need for this directly from the `events.log` file:
+
+```
+Switch FooSwitch  "Foo Switch" {modbus="slave1:0"} 
+Switch BarSwitch  "Bar Switch" {modbus="slave1:1"}
+```
+
+turn into
+
+```
+Switch FooSwitch  "Foo Switch" {channel="modbus:data:wago:wago_slave1:wago_s1_000:switch", autopudate="false"} 
+Switch BarSwitch  "Bar Switch" {channel="modbus:data:wago:wago_slave1:wago_s1_001:switch", autoupdate="false"}
+```
+
+If you have many Items to change and used the naming scheme recommended above, you can now use the following search-and-replace expressions in your editor:
+
+Replace 
+
+`{modbus="slave1:`
+
+by 
+
+`{channel="modbus:data:wago:wago_slave1:wago_s1_00`
+
+in all lines which used single digits for the address in the 1.x config.
+Instead of `wago`, `wago_slave1` and `wago_s1_00` you have to use the names you have chosen for your `Bridge`, `poller` and `data` things.
+Similar expressions are to be used for two-digit and three-digit relative addresses.
+
+Replace 
+
+`"}`
+
+by 
+
+`:switch"}`
+
+in all lines dealing with switches.
+For other Item types use the respective replace strings.
+
+That way you can update even a large amount of Item definitions in only a few steps.
```

Comment:
Do we need those many blank lines here?
The website will make a separate paragraph for each sentence sorrounded by blank lines.

Reply:
Dear @Confectrician 

the idea behind this was to make it easier to copy and paste the strings to the editor. If they were embedded in the text it would make that more difficult in my opinion.